docs/pages/management/dynamic-configuration/spacelift.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: "Manage Dynamic Configuration Resources with Spacelift"
+description: "Learn how to set up Spacelift to manage dynamic configuration resources via GitOps and Teleport's Terraform provider."
+---
+
+You can use Spacelift with Teleport's Terraform provider to manage dynamic
+configuration resources via GitOps and infrastructure as code. This gives you an
+audit trail of changes to your Teleport configuration and a single source of
+truth for operators to examine.
+
+{/*Uncomment when we fix issues with Mermaid support:
+
+```mermaid
+flowchart TB
+subgraph Spacelift
+  idfile["Teleport Identity File"]-->spacelift["Spacelift Worker"]
+end
+
+repo["GitHub Repo"]--"Configuration Resources (Terraform)"--->spacelift
+
+spacelift-->proxy["Teleport Proxy Service"]
+proxy-- "gRPC API Traffic" -->auth["Teleport Auth Service"]
+```
+*/}
+
+<Figure
+  align="center"
+  width="500"
+  height="200"
+>
+  ![Example](../../../img/management/spacelift/spacelift.svg)
+</Figure>
+
+This guide will show you how to set up the GitOps platform Spacelift with the
+Teleport Terraform provider. While following this guide, you will create a no-op
```

Comment:
Do we expect the reader to know the term "no-op", or should we define it?

Reply:
I've made this more explicit. Readers who know what "no-op" means generally might not know what it means in this context